eskip/eskip.go

```diff
@@ -96,6 +96,21 @@ type Route struct {
 	Backend string
 }
 
+type RoutePredicate func(*Route) bool
+
+type RouteList []*Route
```

Comment:
but it wasn't used in other packages as it is only being introduced here. What I mean is that it was only used in the cmd/eskip package, and wasn't part of the public interface.